utils/aws/s3.ts

```diff
@@ -0,0 +1,41 @@
+import aws from "aws-sdk";
+import fs from "fs";
+import path from "path";
+
+const S3 = new aws.S3();
+
+/* 
+
+Uploads a file to the AWS S3 bucket corresponding to AWS_BUCKET_NAME in .env.
+    * filePath: the path of the file to upload
+    * returns: upload response if successful, error object otherwise
+
+Sample usage:
+    myFile = "~/documents/my_file.txt"
+    res = uploadToS3(myFile)
```

Comment:
Have you tried this to make sure it works? it might be helpful to test this if you haven't already on some test endpoint.

Reply:
Yeah, we called the function in an endpoint and got a sample text file to upload